### PR TITLE
feat: Add theme switch

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): #

## PR Type

What kind of change does this PR introduce?

- Feature

## What is the new behavior?

Dark/light theme switch is available

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Dark

image

## Light

image

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.Playground.Shared/Styles/Controls/AppBarButton.xaml [hunk 1]
```diff
@@ -227,7 +227,7 @@
 		</Setter>
 	</win:Style>
 
-	<Style TargetType="AppBarButton"
-		   BasedOn="{StaticResource NativeDefaultAppBarButton}" />
+	<!--<Style TargetType="AppBarButton"
```
Comment (agneszitte-nventive) [inline src/Uno.Playground.Shared/Styles/Controls/AppBarButton.xaml:230]: Can you explain why it's commented here please ;) ? 
Don't need it anymore ? Related to an issue ?

File: src/Uno.Playground.Shared/Styles/Controls/CommandBar.xaml [hunk 2]
```diff
@@ -177,8 +177,8 @@
 		</Setter>
 	</android:Style>
 
-	<!--Default CommandBar Style-->
+	<!--Default CommandBar Style--><!--
```
Comment (agneszitte-nventive) [inline src/Uno.Playground.Shared/Styles/Controls/CommandBar.xaml:180]: Can you explain why it's commented here please ;) ? 
Don't need it anymore ? Related to an issue ?

File: src/Uno.Playground.Shared/Styles/Controls/AppBarButton.xaml (hunk 1 above)
Comment (MartinZikmund) [reply src/Uno.Playground.Shared/Styles/Controls/AppBarButton.xaml:230]: We now support the WinUI AppBarButton, so native style is not needed.

File: src/Uno.Playground.Shared/Styles/Controls/CommandBar.xaml (hunk 2 above)
Comment (MartinZikmund) [reply src/Uno.Playground.Shared/Styles/Controls/CommandBar.xaml:180]: Similar here 👍

Comment (jeromelaban) [reply src/Uno.Playground.Shared/Styles/Controls/CommandBar.xaml:180]: Let's remove it then.

**Review by jeromelaban — APPROVED** [review]

**Review by ajpinedam — APPROVED** [review]

**Pull request merged**